lib/commands/up.ts

```diff
@@ -28,7 +34,7 @@ export const up = async (opts: CommandUpOptions): Promise<void> => {
   try {
     const collection = connection.getMigrationsCollection(migrationsCollection);
     const appliedMigrations = await getAppliedMigrations(collection);
-    const migrations = (await loadMigrations(migrationsDir)).filter(
+    const migrations = (await loadMigrations(migrationsDir, pattern)).filter(
```

Comment:
is there the `glob` argument from the config missing?

Reply:
yes, it's missing. my bad